.changeset/blue-scissors-design.md

```diff
@@ -0,0 +1,5 @@
+---
+'openzeppelin-solidity': minor
+---
+
+`Math`: Make `ceilDiv` to revert on 0 division even if the numerator is 0
```

Comment:
Its actually a panic. But this is already merge so whatever.

Reply:
I think the changeset is correct, it reverts with a Panic error.